Twitch Discord Bridge huge refactoring (#5)
ollprogram/TwitchDiscordBridge: .github/ISSUE_TEMPLATE/refactor_request.md

### Description

> Write here a short description about what should be refactored

Reviewer: ```suggestion
Write a short description about what should be refactored
```

### Explanation

Explain here why and how it should be refactored. 

> Try to explain what are the benefits for the application or for developers.

Reviewer: ```suggestion
Explain how this benefits the application or the developers.
```
